src/App/Formatter/Post/CSV.php

```diff
@@ -125,7 +125,7 @@ private function returnHXLTagAndAttributeString($tag, $hxl_row)
             $tag = '#' . $hxl_row['tag_name'];
         }
         $attribute = $hxl_row['attribute'] && !empty($hxl_row['attribute']) ?
-            ' +'. $hxl_row['attribute'] : '';
+            '+'. $hxl_row['attribute'] : '';
```

Comment:
having an extra space made the preview not be completely correct specifically in the map view of HDX . While HDX was indeed rendering points in the map, they were not always correct (ie first point was wrong)

Reply:
#readthedocs, romina